src/ngraph_builder.cc

```diff
@@ -1368,7 +1430,9 @@ static Status TranslateConv3DOp(
   std::shared_ptr<ng::Node> ng_conv = make_shared<ng::op::Convolution>(
       ng_input, ng_filter, ng_strides, ng_dilations, ng_padding_below,
       ng_padding_above);
-
+  if (!is_ndhwc) {
+    SaveNgOp(first_ng_op_map, op->name(), ng_conv);
+  }
```

Comment:
Q: I understand the the separate cases for _"is_ndhwc"_ and _"!is_ndhwc"_ , but why do we need a third one here: https://github.com/NervanaSystems/ngraph-tf/pull/413/files?utf8=%E2%9C%93&diff=unified&w=1#diff-148e2a1788305495ad1a483e17b53eaaR1419

Reply:
I had actually removed all of them. But i guess i missed saving them. Thanks for pointing them out.